evap/staff/templates/staff_semester_view_evaluation.html

```diff
@@ -189,6 +189,40 @@
             <a href="{% url 'staff:evaluation_copy' evaluation.id %}" class="btn btn-sm btn-light" data-bs-toggle="tooltip" title="{% translate 'Copy' %}">
                 <span class="fas fa-copy"></span>
             </a>
+            {% if not evaluation.has_exam %}
+                <confirmation-modal type="submit" name="evaluation_id" value="{{ evaluation.id }}" confirm-button-class="btn-primary" form="exam_creation_form_{{ evaluation.id }}">
+                    <span slot="title">{% translate 'Create exam evaluation' %}</span>
+                    <span slot="action-text" id="submit_button">{% translate 'Create exam evaluation' %}</span>
+                    <span slot="question">
+                        {% blocktranslate trimmed %}
+                            Create an exam evaluation based on this evaluation. This will copy all the participants and contributors from the original evaluation. It will set the weight of the original evaluation to 9 and its end date will be set to the day before the exam.
+                        {% endblocktranslate %}
+                    </span>
+                    <div slot="extra-inputs">
+                        <label>
+                            {% translate 'Exam Date:' %}
+                            <input type="date" name="exam_date" data-earliest-legal-exam-evaluation-date="{{ evaluation.vote_start_datetime }}" class="form-control" required form="exam_creation_form_{{ evaluation.id }}"/>
+                        </label>
+                    </div>
+
+                    <button slot="show-button" type="button" class="btn btn-sm btn-light" title="{% translate 'Create exam evaluation' %}" data-bs-placement="top" data-bs-toggle="tooltip"><span class="fas fa-file-pen fa-fw"></span></button>
+
+                    <script type="text/javascript">
+                        document.querySelectorAll("[data-evaluation-date]").forEach(element => {
+                            element.addEventListener("change", () => {
+                                if (element.value <= element.dataset.earliestLegalExamEvaluationDate) {
+                                    element.setCustomValidity("{% translate 'The exam date is before the start date of the main evaluation.' %}");
```

Comment:
This isn't the semantics I intended with the naming "earliest legal".

If "earliest legal day" is 01.01.2010, then the user is allowed to choose 01.01.2010.